deploy/operator/provision-shared-host-macos.yaml

```
@@ -0,0 +1,108 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: provision-shared-host-macos
```

We are not provisioning a shared host, right? 

```suggestion
  name: provision-host-macos
```